corehq/apps/translations/app_translations/upload_module.py

```diff
@@ -140,6 +144,20 @@ def _update_case_search_labels(self, rows):
                     self.msgs.append((messages.error, message))
                     continue
                 self._update_translation(display_row, prop.label)
+        else:
+            message = _(
+                'Expected {expected_count} case_search_display '
+                'properties in  menu {index}, found {actual_label_count} for case_search_display. '
+                'Case Search config properties for menu {index} were only partially updated.'
+            ).format(
+                expected_count=len(properties),
+                actual_label_count=len(displays),
+                index=self.module.id + 1,
+            )
+            self.msgs.append((messages.warning, message))
+            partially_update_translations(displays, is_display=True)
+
+        if len(hints) == len(properties):
```

Comment:
Recognizing this wasn't part of your changes, but this message could be consolidated with the almost exact same one for display
```
'A hint row for menu {index} has an unexpected case search property "{field}". '
                                'Case properties must appear in the same order as they do in the bulk '
                                'app translation download. No translations updated for this row.'

Reply:
my response to your other comment also applies to this one!

Reply:
I'm seeing that string is already used in two places, but one says "A hint row" while the other is "A display row"

I was thinking we can do something like
```
msg = 'A {row_property} row for menu {index} has an unexpected case search property "{field}". '
                                'Case properties must appear in the same order as they do in the bulk '
                                'app translation download. No translations updated for this row.
```

and we can do msg.format('hint'...) and msg.format('display'...). But nbd, certainly not blocking.

Reply:
ah you're right I misread the part of the code this comment was attached to - made the error message reusable!